roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -153,6 +153,8 @@ virt_type={{ nova.libvirt_type }}
 {% if nova.libvirt_cpu_model -%}
 cpu_mode = custom
 cpu_model = {{ nova.libvirt_cpu_model }}
+{% elif ansible_architecture == "ppc64le" %}
+cpu_mode = host-passthrough
```

Comment:
Looks like if we do ever define a cpu_model, we'll have to do it at a group_vars level lower than `all` since they'll need to differ between x86 and non. Just a thing ot remember.